webclient/index.html

```diff
@@ -681,6 +685,35 @@
         link.click();
     }
 
+
+
+    const configureAuthClient = async ()=>{
+
+        const config = {domain: "{{ .Env.AUTH0_DOMAIN }}", clientId: "{{ .Env.AUTH0_CLIENT_ID }}"};
+
+        auth0Client =  await auth0.createAuth0Client({
+            domain: config.domain,
+            clientId: config.clientId,
+            cacheLocation: 'localstorage'
+        });
```

Comment:
@UtopicUnicorn thank you for your contribution💪 

**(issue)** Could you please add exception handling: let's fall back on the "incognito" role (no authN) to avoid complete UX disruption.